packages/storybook-css/src/LinkList.tsx

```diff
@@ -9,7 +9,6 @@ import { Link } from '@utrecht/component-library-react';
 import { UtrechtIconArrow } from '@utrecht/web-component-library-react';
 import clsx from 'clsx';
 import React, { HTMLAttributes, PropsWithChildren, ReactNode } from 'react';
-import '@utrecht/components/link-list/css/index.scss';
 
```

Comment:
Hmm interesting I would expect this CSS to be needed still? 

However this story should probably be refactored anyway and use the React LinkList component. 🤔